_pages/home/updating-to-version-2/updating-to-version-2.md

```diff
@@ -21,6 +21,8 @@ To update to version 2.0 of Spartacus, run the following command in the workspac
 ng update @spartacus/schematics
 ```
 
+> NOTE: in case you don't have `@spartacus/schematics` installed, you'll get the following error: `Package '@spartacus/schematics' is not a dependency.`. Before you can use the update schematics, you need to add `@spartacus/schematics` to your `package.json` (make sure that the version matches the other spartacus libraries) and install it using your package manager (npm or yarn).
+
```

Comment:
It's nice that we provide the error message right here so users don't get stuck.  That would have helped me.

For in schematics installation: I feel we're still missing key information on how to install schematics.  Is it just one command? If so, can we just give it to the user `ng add @spartacus/schematics`.  If it's more complex than that, we could link to an installation instruction section in the schematics docs.

I don't understand what this part means: `(make sure that the version matches the other spartacus libraries)`.  What I don't understand is:
* Which versions? : The ones I already have?  The ones I want to upgrade to?  
* How do I make sure of that?

Reply:
Discussed offline